### ISSUE TITLE
Bug in support of 'target' attribute

### ISSUE THREAD

**Issue by geonanorch** [riot/route#149]
Just found a small bug in v5.3.2, which results in 'target' attributes being ignored. In:

```javascript
const isForbiddenLink = el => !el || !isLinkNode(el) // not A tag
|| has(el, DOWNLOAD_LINK_ATTRIBUTE) // has download attr
|| !has(el, HREF_LINK_ATTRIBUTE) // has no href attr
|| isTargetSelfLink(el.target) || isCrossOriginLink(el.href);
```

the call `isTargetSelfLink(el.target)` should read `isTargetSelfLink(el)`

**Comment by GianlucaGuarini** [comment]
@geonanorch thank you. Can you please open a PR for it?

**Comment by geonanorch** [comment]
Sure, though threre is really not much to it, more like a typo. [PR#150](https://github.com/riot/route/pull/150).

**Comment by GianlucaGuarini** [comment]
thank you. It's fixed in @riotjs/route@5.3.3
